src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -576,6 +592,24 @@ describe('manage nsfs cli account flow', () => {
             expect(new_account_details.allow_bucket_creation).toBe(true);
         });
 
+        it('cli account update account by name - check for the symlink relative path, not absolute path', async function() {
+            const { name } = defaults;
+            const new_name = 'account1_new_name';
+            const access_key = 'GIGiFAnjaaE7OKD5N123';
+            const secret_key = 'U3AYaMpU3zRDcRFWmvzgQr9MoHIAsD+3oEXAM123';
```

Comment:
Are you updating the account name and also the `access_key` and `secret_key`?
If so you can edit the test name to reflect this change.

Regarding `secret_key` we add the suffix of `EXAMPLE` to avoid automatic alerts of secret leaking if you can please use it also.